integration/port_forwarding_test.go

```diff
@@ -145,14 +146,18 @@ func testPortForwarding(t *testing.T, suite *integrationTestSuite) {
 			cl.Stdout = term
 			cl.Stdin = term
 
-			sshSessionCtx, sshSessionCancel := context.WithCancel(context.Background())
+			sshSessionCtx, sshSessionCancel := context.WithCancel(ctx)
 			go cl.SSH(sshSessionCtx, []string{}, false)
 			defer sshSessionCancel()
 
-			timeout, cancel := context.WithTimeout(context.Background(), 5*time.Second)
-			defer cancel()
-			_, err = waitForSessionToBeEstablished(timeout, apidefaults.Namespace, site)
-			require.NoError(t, err)
+			sessionEstablished := func() bool {
```

Comment:
Can you add a comment here that explains why we aren't using `waitForSessionToBeEstablished` just to prevent someone from reintroducing the issue?

Reply:
Good point, I've replaced `waitForSessionToBeEstablished` with the preferable session tracker logic instead.